oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -118,7 +119,7 @@ public String getPath() {
 
     @Override
     public String getCommandLine() {
-        return commandLine.get();
+       return Arrays.stream(commandLine.get().split("\0")).collect(Collectors.joining(" "));
```

Comment:
Since the commandline won't change as long as the process is active, this parsing can be done in the memoized `queryCommandLine()` method so it only has to be done once.